### PR TITLE
test commit

### PR DESCRIPTION
**Pull request by Jim-Dillon**
test

### CODE REVIEW

File: src/js/dropdown.js [hunk 1]
```diff
@@ -6,3 +6,6 @@ const addDropdown = () => {
 const removeDropdown = () => {
     document.getElementsByClassName('dropdown-header').classList.remove('dropdown-header');
 }
+
+
+
```
Comment (anthonydillon) [inline src/js/dropdown.js]: I think you should remove these lines

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Comment in the code

**Comment by Jim-Dillon** [comment]
thanks addressed the comments - ready for re-review

**Review by anthonydillon — APPROVED** [review]
LGTM :+1:

**Pull request merged**